app/views/layouts/application.html.erb

```diff
@@ -5,11 +5,14 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
   </head>
 
   <body>
-    <%= yield %>
+    <main class="container mx-auto mt-28 px-5 flex">
```

Comment:
Did you check to see if the styles applied to the `/trips` page? Can you post a screenshot

Reply:
Happy to help! I have not applied any styles to the /trips page. I am actually working on that right now.

I am happy to post a screenshot once done!

Reply:
Since you added the tags to application.html.web, I think the trips page will get styling automatically

Reply:
So you should be able to fire up the app on this branch and visit /trips and see newer styles

Reply:
I could be wrong though lol

Reply:
![image](https://github.com/RossaMania/PushPin/assets/98660173/af76ae59-be0b-4690-a1b0-d46ba5553763)

Happy to help! Here is a screenshot of the Tailwind CSS test code in my VS Code, and the "Hello World!" statement at http://localhost:3000/trips for your viewing pleasure.